### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Liam0**
No Cash/license update when you purchase license the actual variable for
license were set after the sync, Sorry about that jawshy.
Press escape to get rid of the annoying storagebox.... objNull!
Confirmation before selling vehicles out of the garage using
displayName.
Garage text out of place !SPACES!

### CODE REVIEW

File: Altis_Life.Altis/dialog/impound.hpp [hunk 1]
```diff
@@ -76,7 +76,7 @@ class Life_impound_menu {
         class SellCar: Life_RscButtonMenu {
             idc = -1;
             text = "$STR_Global_Sell";
-            onButtonClick = "[] call life_fnc_sellGarage; closeDialog 0;";
+            onButtonClick = "[] spawn life_fnc_sellGarage;";
```
Comment (BoGuu) [inline Altis_Life.Altis/dialog/impound.hpp]: Why the spawn?

Comment (Liam0) [inline Altis_Life.Altis/dialog/impound.hpp]: https://community.bistudio.com/wiki/BIS_fnc_guiMessage

Comment (BoGuu) [inline Altis_Life.Altis/dialog/impound.hpp]: My bad, passed my mind that it would be unscheduled

**Comment by danielstuart14** [comment]
Ready for merging?

**Comment by Liam0** [comment]
Only if you want it in, should be fine unless anyone have something i missed

File: Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf [hunk 2]
```diff
@@ -56,15 +56,15 @@ if (!(_sellPrice isEqualType 0) || _sellPrice < 1) then {_sellPrice = 500;};
 if (!(_retrievePrice isEqualType 0) || _retrievePrice < 1) then {_retrievePrice = 500;};
 
 (CONTROL(2800,2803)) ctrlSetStructuredText parseText format[
-    (localize "STR_Shop_Veh_UI_RetrievalP")+ " <t color='#8cff9b'>$%1</t><br/>
```
Comment (tkcjesse) [inline Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf]: Why remove the ident? We should have lines 59-75 indented for readability.

Comment (Liam0) [inline Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf]: https://camo.githubusercontent.com/db493b1fc14694f4720364537e42975a8066a8be/687474703a2f2f7075752e73682f70697367442f643264626233643764362e706e67

Comment (danielstuart14) [inline Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf]: first line is missing the space.

Comment (Liam0) [inline Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf]: Still makes it look silly ingame with floating text in the middle of the frame.

**Pull request closed without merging**